book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] We forgot to add a many-to-many relationship between `Book` and `Person` Which should be through `Rental` so `Rental` has a 1-to-many relationship with `Book` here we need to add a one-to-many relationship by defining a function for that
```ruby
def add_rental(person,date)
  Rental.new(person,date,self)
 end
 ```